packages/thirdweb/src/wallets/coinbase/coinbaseSDKWallet.ts

```diff
@@ -235,6 +232,7 @@ function onConnect(
         address: getAddress(accounts[0]),
       };
       emitter.emit("accountChanged", newAccount);
+      emitter.emit("accountsChanged", accounts);
```

Comment:
just a comment, these are pretty close but do pretty different things, I wonder if we need the `accountChanged`

not a blocker but we should revisit this